### ISSUE TITLE
Web pages language doesn’t match user preferred locale

### ISSUE THREAD

**Issue by TheoChevalier** [mozilla-mobile/focus-android#4154]
## Steps to reproduce

Use Android in French (or select French in the Focus language settings), then open `mozilla.org`. The issue happen with both the locale set to “Use OS preferred locale” or explicitly set to French.

### Expected behavior

Web page language matches UI locale, and `mozilla.org` redirects to `mozilla.org/fr/`

### Actual behavior

`mozilla.org` redirects to `mozilla.org/en-US/`

### Device information

* Android device: `Sony Xperia Z3C, Android 6.0.1`
* Focus version: `9.0 (Build #330171230 🦎 64.0.20190111134233)`

**Comment by Delphine** [comment]
I confirm this happens to me as well on my Pixel 2 (running Android v9), on Focus version: 9.0 (Build #330221220 🦎 65.0.20190114172331)

**Comment by lobontiumira** [comment]
This is still reproducible on the 8.16.0 build with Samsung Galaxy Note 8 (Android 9).

**Comment by Delphine** [comment]
@TheoChevalier Is this still reproducible for you?
I get to mozilla.org/fr now from my Pixel 2 device, both when I set the language to French from the browser explicitly - and when I choose the “Use OS preferred locale”.

**Comment by TheoChevalier** [comment]
Unfortunately this Android device stopped working, so I can’t tell 😬

**Comment by lobontiumira** [comment]
I was able to reproduce this issue on Focus Nightly 97.0 (build 353631136 GV 97.0a1-20211228034259) with Sony Xperia Z5 Premium (Android 7.1.1), in both situations:
* setting the devce to French, and
* setting Focus to French.

On Google Pixel (Android 10), on the same Focus Nightly version from 12/29 I was able to reproduce the issue only when setting Focus to French. When I set the device to French, mozilla.org/fr was loaded.
